source/exploring_catalog_and_datasets/03_searching_the_data/search.rst

```diff
@@ -137,6 +137,8 @@ your queries to filter the results based on a specific field's value.
      * The language of the dataset (iso code)
    * * theme
      * The theme of the dataset
+   * * keyword
+     * The keywords describing the dataset
```

Comment:
`A keyword describing the dataset` ?

Keyword is multivalued but theme is multivalued too ...

Reply:
Suggestion implemented in 313dd03